pkg/aws/eni/node_manager_test.go

```diff
@@ -459,46 +459,46 @@ func (e *ENISuite) TestNodeManagerReleaseAddress(c *check.C) {
 	c.Assert(mngr, check.Not(check.IsNil))
 
 	// Announce node, wait for IPs to become available
-	cn := newCiliumNode("node3", "i-testNodeManagerReleaseAddress-1", "m4.xlarge", "us-west-1", "vpc-1", 1, 4, 10, 0)
```

Comment:
Actually, while backporting this to v1.9 I noticed that this line here silently changed FirstInterfaceIndex to 0, while it should be 1 on v1.8
```suggestion
	cn := newCiliumNode("node3", "i-testNodeManagerReleaseAddress-1", "m4.xlarge", "us-west-1", "vpc-1", 1, 2, 10, 0)
```